orchagent/copporch.cpp

```diff
@@ -447,7 +510,12 @@ void CoppOrch::doTask(Consumer &consumer)
         }
         catch(const std::out_of_range e)
         {
-            SWSS_LOG_ERROR("processing copp rule threw exception:%s", e.what());
+            SWSS_LOG_ERROR("processing copp rule threw exception:%s, line:%s", e.what(), __LINE__);
```

Comment:
@hrachya-mellanox ,please address kamil's comments in your previous pull request.